site/docs/blogs.md

```diff
@@ -21,6 +21,15 @@ title: "Blogs"
 ## Iceberg Blogs
 
 Here is a list of company blogs that talk about Iceberg. The blogs are ordered from most recent to oldest.
+<!-- markdown-link-check-disable-next-line -->
+### [Athena vs. Snowflake on Iceberg: Performance and Cost Comparison on TPC-H](https://www.ryft.io/blog/athena-vs-snowflake-on-iceberg-tpch-comparison)
+**Date:** July 9, 2025, **Company**: Ryft
+**Author**: [Yuval Yogev](https://www.linkedin.com/in/yuvalyogev)
```

Comment:
@yogevyuval I don't think we should include this blog as it's directly comparing vendor solutions as opposed to providing informational/educational content about Iceberg or related use cases.  Hosting blogs like this may give a false impression that the project is endorsing one solution over another or agrees in some way with the methodology or results of the comparison.

Reply:
@danielcweeks Got it - the intention of the blog is only to share technical material, but i understand it - removing this one